docs/dev/docker-hub.md

```diff
@@ -157,13 +157,15 @@ The ScyllaDB image supports many command line options that are passed to the `do
 
 #### `--seeds SEEDS`
 
-The `-seeds` command line option configures ScyllaDB's seed nodes.
-If no `--seeds` option is specified, ScyllaDB uses its own IP address as the seed.
+The `--seeds` command line option configures ScyllaDB's seed node.
+To start a single-node ScyllaDB cluster, pass the localhost address (127.0.0.1) as the `--seeds` argument.
+If no `--seeds` option is specified and both the listen address and broadcast address are equal to the localhost address
+or the `--developer-mode=1` option is used, ScyllaDB automatically uses the localhost address as the seed.
```

Comment:
I can't tell if this is a behavioral change or not.

If it is, don't we have to update lots of examples?

Reply:
It is a behavioral change but for an under-documented feature.
We have examples for providing `seeds`
But where are those examples for provding no `seeds`?